contracts/lbp/LBPManager.sol

```diff
@@ -126,80 +121,70 @@ contract LBPManager {
     }
 
     /**
-     * @dev                             Subtracts the primeDaoFee and adds liquidity to the LBP.
-     * @param _tokenList                Numerically sorted array (ascending) containing two addresses:
-                                            - The address of the project token being distributed.
-                                            - The address of the funding token being exchanged for the project token.
+     * @dev                             Subtracts the fee and adds liquidity to the LBP.
      * @param _projectTokenIndex        Index for the _tokenList array for the funding token.
      * @param _sender                   Address of the liquidity provider.
      * @param _userData                 UserData object that specifies the type of join.
      */
     function addLiquidity(
-        IERC20[] memory _tokenList,
         uint8 _projectTokenIndex,
         address _sender,
         bytes memory _userData
     ) external onlyAdmin {
-        require(!poolFunded, "LBPManager: pool has already been funded");
+        require(!poolFunded, "LBPManager: pool already funded");
         poolFunded = true;
+        projectTokenIndex = _projectTokenIndex;
```

Comment:
why do we need to assign `_projectTokenIndex` value to a new variable here?

Reply:
I did this because it was a comment in the PR #150 from @fabianschu. For the implementation the way it is now it only adds the benefit of removing the `_projectTokenIndex` parameter from the functions.

Reply:
Through the PR that will be created form issue #143 we will get the `projectTokenIndex` from the `_tokenLIst` array. I can change it here to just use the parameter?

Reply:
ok, let's do it later in #143